google/cloud/sql/connector/instance_connection_manager.py

```diff
@@ -287,6 +314,32 @@ def _auth_init(self) -> None:
 
         self._credentials = credentials
 
+    async def seconds_until_refresh(self) -> int:
```

Comment:
Same as in Java - I think this can be a field that is set before the refresh's result is returned. It should be both calculated once (per result), and specific to a refresh result.

Reply:
This is now being called in `get_instance_data` and is passed in as an attribute of `InstanceData`